azure-iot-device/azure/iot/device/common/pipeline/pipeline_ops_mqtt.py

```diff
@@ -6,7 +6,7 @@
 from . import PipelineOperation
 
 
-class SetConnectionArgs(PipelineOperation):
+class SetMQTTConnectionArgsOperation(PipelineOperation):
```

Comment:
>MQTT [](start = 9, length = 4)

I think we need to be consistent here - either all the MQTT pipeline ops have MQTT in the name, or none of them do.

Personally I lean towards all of them do, for the sake of test output.

e.g. MQTTPublishOperation